inc/zoo/meta/BitmaskMaker.h

```diff
@@ -40,8 +40,21 @@ struct BitmaskMaker {
         >::value;
 };
 
+/// Provides TopBlit to clear any garbage bits at the top of a bitmask made by BitmaskMaker
+template<typename T, int CurrentSize>
+struct BitmaskMakerClearTop {
+    // To ensure 0s above CurrentSize
+    constexpr static T Eight = 8;
```

Comment:
Come on?

Reply:
I got very frustrated with int promotion for a moment and forgot to clean up after. :D